cupix/cupix.egg-info/PKG-INFO

```diff
@@ -1,8 +1,8 @@
-Metadata-Version: 0.0
+Metadata-Version: 2.4
 Name: cupix
 Version: 2025.0.0
 Summary: Likelihood for Lyman-alpha cross-power spectrum
-Author: Martine Lokken, Jonas Chaves-Montero, Andreu Font-Ribera, Chris Pedersen, Sindhu Satyavolu
+Author: Chris Pedersen
```

Comment:
This looks like a mistake? I don't think we want to say that the author is Chris Pedersen :-)

Reply:
Is it possible that you have added the wrong file by mistake? I see that you have also deleted other info below